### DISCUSSION TITLE
Persisting a database to disk not working for me

### DISCUSSION THREAD

**Discussion by mrMetalWood** [duckdb/duckdb#10861] [Q&A]
I installed duckdb (v0.10.0) via homebrew on a completely new/fresh Mac Book Pro.

It all works fine when using the "in-memory" mode, but when trying to persist the db on the disk I always get the error:

`Error: INTERNAL Error: Attempted to dereference unique_ptr that is NULL!`

![Screenshot 2024-02-27 at 11 50 07](https://github.com/duckdb/duckdb/assets/2902795/b2ec9d53-f159-4bf7-a53b-5abc632e4120)

I also had the problem on another machine. Does anybody else experience this? Or am I missing something obvious here? 🙈

**Comment by Tishj** [comment]
Can you raise an issue instead, InternalExceptions are always a bug, we would love it if you can supply the problematic parquet file so we can reproduce it

**Reply by mrMetalWood** [reply]
Sure :) I filed an issue #10871
